shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -557,17 +557,20 @@ private void flushAllViews() {
     vdControllers.clear();
   }
 
-  public void onDisplayPlatformView(int viewId, int x, int y, int width, int height) {
-    // TODO: Implement this method. https://github.com/flutter/flutter/issues/58288
-  }
-
-  public void onDisplayOverlaySurface(int id, int x, int y, int width, int height) {
-    FlutterView flutterView = (FlutterView) this.flutterView;
+  private void initializeRootImageViewIfNeeded() {
     if (!flutterViewConvertedToImageView) {
       flutterView.convertToImageView();
       flutterViewConvertedToImageView = true;
     }
+  }
 
+  public void onDisplayPlatformView(int viewId, int x, int y, int width, int height) {
+    initializeRootImageViewIfNeeded();
+    // TODO: Implement this method. https://github.com/flutter/flutter/issues/58288
+  }
+
+  public void onDisplayOverlaySurface(int id, int x, int y, int width, int height) {
+    initializeRootImageViewIfNeeded();
```

Comment:
This one is redundant, perhaps it can be removed, but it doesn't hurt to do it in both places.